COCO_DATA_FORMAT.md

```diff
@@ -252,3 +252,32 @@ The dataset format is a simple variation of COCO, where `image_id` of an annotat
     ]
 }
 ```
+
+Another example for object detection:
+```json
+{
+    "images": [
+        {
+            "id": 1,
+            "width": 224,
+            "height": 224,
+            "file_name": "1.jpg",
+            "zip_file": "test.zip"
+        }
+    ],
+    "annotations": [
+        {
+            "id": 1,
+            "image_ids": [1],
+            "key_value_pairs": {
```

Comment:
Just noticed this, should we rename this to match the API terminology (`fieldSchema` for the schema and `fields` in the response)?